### PR TITLE
Save

### PR DESCRIPTION
**Pull request by mikekestemont**
This solves a bug when loading a trained model.

### CODE REVIEW

File: pandora/impl/keras/model.py [hunk 1]
```diff
@@ -312,7 +312,7 @@ def load(model_dir,
              batch_size=50):
         # load model and weights:
         with open(os.path.join(model_dir, 'model_architecture.json'), 'r') as f:
-            model = model_from_json(f)
+            model = model_from_json(f.read())
```
Comment (PonteIneptique) [inline pandora/impl/keras/model.py:315]: As per 
> You can then build a fresh model from this data:

```python
# model reconstruction from JSON:
from keras.models import model_from_json
model = model_from_json(json_string)
```
from 
https://keras.io/getting-started/faq/

**Review by PonteIneptique — APPROVED** [review]

**Review by PonteIneptique — APPROVED** [review]

**Comment by mikekestemont** [comment]
watch out: that new strategy does not work for all types of models.

**Comment by PonteIneptique** [comment]
Can you describe when it should not work ? Version of keras issue ?

**Comment by mikekestemont** [comment]
Whenever you have custom layers it tends to break.

**Pull request merged**